src/components/va-alert/va-alert.css

```diff
@@ -0,0 +1,142 @@
+@import '../../mixins/accessibility.css';
+
+:host {
+  display: block;
+  background-color: var(--color-gray-lightest);
+  font-size: 2rem;
+  position: relative;
+}
+
+:host([background-only='true']),
+:host([background-only='']) {
+  background-color: unset;
+}
+
+div.alert {
+  display: table;
+  padding: 3.2rem 6.4rem 3.2rem 2.4rem;
+  width: 100%;
+  vertical-align: middle;
+  border-left-style: solid;
+  border-left-width: 10px;
+  box-sizing: border-box;
+}
+
+div.alert > i::before {
+  display: flex;
+  margin-right: 1.6rem;
+}
+
+div.body {
+  padding-left: 0;
+  padding-right: 0;
+  display: table-cell;
+  vertical-align: middle;
+  width: 100%;
+}
+div.text {
+  margin-top: 2rem;
+}
+.bg-only .text {
+  margin-top: 0;
+}
+
+div.info > i::before {
+  content: '\F05A';
+}
+
+div.continue > i::before {
+  content: '\F023';
+  color: var(--color-green);
+}
+
+div.success > i::before {
+  content: '\F00C';
+  color: var(--color-green);
+}
+
+div.warning > i::before {
+  content: '\F071';
+}
+
+div.error > i::before {
+  content: '\F06A';
+  color: var(--color-secondary-dark);
+}
+
+div.alert.bg-only > i::before {
+  content: '';
+}
+
+div.bg-only {
+  border: none;
+}
+
+.info {
+  border-left-color: var(--color-primary-alt-dark);
+}
+.info.bg-only {
+  background-color: var(--color-primary-alt-lightest);
+}
+
+.continue,
+.success {
+  border-left-color: var(--color-green);
+}
+
+.continue.bg-only {
+  background-color: var(--color-gray-lightest);
+}
+.success.bg-only {
+  background-color: var(--color-green-lightest);
+}
+
+.warning {
+  border-left-color: var(--color-gold);
+}
+
+.warning.bg-only {
+  background-color: var(--color-gold-lightest);
+}
+
+.error {
+  border-left-color: var(--color-secondary-dark);
+}
+
+.error.bg-only {
+  background-color: var(--color-secondary-lightest);
+}
+
+.va-alert-close {
+  margin: 1.6rem;
+  padding: 0;
+  width: auto;
+  color: var(--color-primary);
+  font-size: 2.25rem;
+  appearance: none;
+  border: 0;
+  cursor: pointer;
+  background: transparent;
+  display: block;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+.va-alert-close:hover {
+  color: var(--color-primary-darker);
+}
+
+i {
+  font-family: 'Font Awesome 5 Free';
+  font-style: normal;
+  font-weight: 900;
+}
+i.fa-times-circle::before {
+  content: '\F057'; /* fa-times-circle*/
+}
+
+h3 {
+  line-height: 26px;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Well this seems oddly specific. Copy pasta?

Reply:
Yup - `uswds` is removing padding and margin from the headings:

https://github.com/uswds/uswds/blob/43f8f697e7924ccaae26654f815fcf4860a5ac15/src/stylesheets/components/_alerts.scss#L85-L88